subprojects/ide-provisioning-api/src/main/java/org/gradle/profiler/ide/CommandChain.java

```diff
@@ -0,0 +1,8 @@
+package org.gradle.profiler.ide;
+
+public interface CommandChain {
```

Comment:
🤔 This is looks like leaked to API implementation details. Not sure that it's reasonable for API client to manipulate `Command` terms for running an IDE. Client wants just to invoke `fun sync(project, ide)`, probably add some VM arguments, but that's it.

WDYT?